alliance_auth/urls.py

```diff
@@ -203,4 +204,18 @@
                        url(r'^notifications/$', 'notifications.views.notification_list', name='auth_notification_list'),
                        url(r'^notifications/(\w+)/$', 'notifications.views.notification_view', name='auth_notification_view'),
                        url(r'^remove_notifications/(\w+)/$', 'notifications.views.remove_notification', name='auth_remove_notification'),
-)
+
+                       # FleetActivityTracking (FAT)
+                       url(r'^fat/$', 'fleetactivitytracking.views.fatlink_view', name='auth_fatlink_view'),
```

Comment:
Actually I'm all for changing it if someone have an idea of name and abbreviation. For *some* reason "Space Vehicle Event Attendance", or SVEA for short, was not liked when I suggested it in alliance. Thought it was too like our corp name, can't imagine why ;)

But jokes aside, if you have a better name suggestion I can deferment definitely go through and change it.